### PR TITLE
Proofread updated Polkit chapter

### PR DESCRIPTION
**Pull request by cjschroder**
some punctuation and spelling corrections, correct title case to lowercase, add title and xml:id to procedure

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP4/openSUSE Leap 15.4
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [x] SLE 15 SP2/openSUSE Leap 15.2
  - [x] SLE 15 SP1
  - [x] SLE 15 SP0
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by cjschroder** [comment]
@mgerstner Will you review, please? There are some light wording edits, punctuation, spelling, and some XML and entity corrections. 

This is such a great update, thanks again. I learned stuff.

File: xml/security_polkit.xml [hunk 1]
```diff
@@ -134,7 +145,7 @@
      </term>
      <listitem>
       <para>
-       Allows to execute programs as a different user based on &pk; authorization
+       Allows excuting programs as a different user based on &pk; authorization
```
Comment (mgerstner) [inline xml/security_polkit.xml:148]: `excuting` -> `executing`

File: xml/security_polkit.xml [hunk 2]
```diff
@@ -364,7 +380,7 @@ org.freedesktop.login1.reboot:
    <para>
     The file
     <filename>/usr/share/polkit-1/actions/org.opensuse.policykit.gparted.policy</filename>
-    has the following content:
+    contains the following content:
```
Comment (mgerstner) [inline xml/security_polkit.xml:383]: `contains the following content` sounds repetitive / redundant to me, but maybe it's just me ...

**Review by mgerstner — COMMENTED** [review]
Content wise everything is still correct as far as I can see. I added two minor comments. Thanks again for taking over this PR

File: xml/security_polkit.xml (hunk 2 above)
Comment (cjschroder) [reply xml/security_polkit.xml:383]: It does sound repetitive. Reverted.

**Comment by cjschroder** [comment]
Thank you for the fast review! Merging.

**Pull request merged**